cli/dts/lib.deno.ns.d.ts

```diff
@@ -163,6 +163,26 @@ declare namespace Deno {
    * */
   export function test(name: string, fn: () => void | Promise<void>): void;
 
+  /** Register a test which will be run when `deno test` is used on the command
+   * line and the containing module looks like a test module.
+   * `fn` can be async if required.
+   *
+   * ```ts
+   * import {assert, fail, assertEquals} from "https://deno.land/std/testing/asserts.ts";
+   * 
+   * Deno.test("My test description", ():void => {
+   *   assertEquals("hello", "hello");
+   * }, { only: true });
+   * 
+   * Deno.test("My async test description", async ():Promise<void> => {
+   *   const decoder = new TextDecoder("utf-8");
+   *   const data = await Deno.readFile("hello_world.txt");
+   *   assertEquals(decoder.decode(data), "Hello world");
+   * }, { only: true });
+   * ```
+   */
+  export function test(name: string, fn: () => void | Promise<void>, options?: Omit<TestDefinition, "name" | "fn">): void;
```

Comment:
Shouldn't this *replace* the old overload on line 164? Because the `options` argument is optional, this overload also matches the two-argument case.

Reply:
Hi @solson , I'm considering keep it because maybe the runtime will be better for those who doesn't use this `options` argument.
It won't make much difference, but every small runtime reduce helps. What do you say?